src/Dapr.Actors/Runtime/ActorReminder.cs

```diff
@@ -124,6 +124,34 @@ internal ActorReminder(ActorReminderOptions options)
             this.Period = options.Period;
             this.Ttl = options.Ttl;
         }
+        
+        /// <summary>
+        /// Initializes a new instance of <see cref="ActorReminder" />.
+        /// </summary>
+        /// <param name="actorType">The actor type.</param>
+        /// <param name="actorId">The actor id.</param>
+        /// <param name="name">The reminder name.</param>
+        /// <param name="state">The state associated with the reminder.</param>
+        /// <param name="dueTime">The reminder due time.</param>
+        /// <param name="period">The reminder period.</param>
+        /// <param name="repetitions">The number of times reminder should be invoked.</param>
+        public ActorReminder(
+            string actorType,
+            ActorId actorId,
+            string name,
+            byte[] state,
+            TimeSpan dueTime,
+            TimeSpan period,
+            int repetitions)
+            : this(actorType, actorId, name, state, dueTime, period)
+        {
+            if (repetitions == 0 || repetitions < -1)
+            {
+                throw new ArgumentException(String.Format("{0} must be a valid value", repetitions),
```

Comment:
Exception should include the value range since we know it.

Reply:
Fixed